h2o-core/src/main/java/hex/grid/HyperSpaceWalker.java

```diff
@@ -106,6 +106,32 @@
    */
   MP getParams();
 
+  /**
+   * Return estimated grid work.
+   * Can return Long.MAX_VALUE if no estimate is available.
+   * @param maxModels
+   * @return estimate of grid work
+   */
+  default long getGridWork(long maxModels) {
```

Comment:
nitpick: let's name it for what it is, `estimateGridWork` (?) and keep `get__` for property getters.